src/components/scenes/Fio/FioStakingChangeScene.tsx

```diff
@@ -295,9 +297,11 @@ export const FioStakingChangeScene = withWallet((props: Props) => {
           <ModalMessage>{lstrings.staking_change_explaner1}</ModalMessage>
           <ModalMessage>{lstrings.staking_change_explaner2}</ModalMessage>
         </View>
-        <RowUi4 rightButtonType="editable" title={lstrings.staking_change_add_amount_title} onPress={handleAmount}>
-          <EdgeText style={styles.amountText}>{exchangeAmount}</EdgeText>
-        </RowUi4>
+        <CardUi4 marginRem={1}>
```

Comment:
Explainer already has 0.5 rem margins per UI4 expectations, why are we adding extra here?

Reply:
The card is not in the explainer view so it needs to have margins added itself.